pkg/plugins/platform/utils.go

```diff
@@ -145,6 +145,20 @@ func Cleanup(app App) error {
 		}
 	}
 
+	volumes, err := client.ListVolumes(docker.ListVolumesOptions{})
+	if err != nil {
+		return err
+	}
+
+	for _, volume := range volumes {
+		if volume.Labels["com.docker.compose.project"] == "ddev"+strings.ToLower(app.GetName()) {
+			err := client.RemoveVolume(volume.Name)
+			if err != nil {
+				return fmt.Errorf("could not remove volume %s: %v", volume.Name, err)
+			}
+		}
+	}
```

Comment:
There is the volume for nginx logs, and it's possible containers from other services might define volumes to be cleaned up as well. I probably wouldn't have implemented this for this PR on its own, but since the effort was already made in the PR for the volume-based approach, I figured it didn't hurt to port the code.